src/chatdDb.h

```diff
@@ -131,8 +131,8 @@ class ChatdSqliteDb: public chatd::DbInterface
     virtual int updateSendingItemsMsgidAndOpcode(karere::Id msgxid, karere::Id msgid)
     {
         mDb.query(
-            "update sending set opcode=?, msgid=? where chatid=? and opcode=? and msgid=?",
-            chatd::OP_MSGUPD, msgid, mChat.chatId(), chatd::OP_MSGUPDX, msgxid);
+            "update sending set opcode=?, msgid=? where chatid=? and (opcode=? or opcode=?) and msgid=?",
```

Comment:
I think this is wrong. Since we are updating the `msgid`, it means we already received the `NEWMSGID`, so no `NEWMSG` should be in the sending table anymore. It should have been removed from a previous call to `deleteSendingItem()` (from `msgRemoveFromSending()`).